lib/parsec/jinja2support.py

```diff
@@ -93,9 +94,14 @@ def jinja2environment(dir_=None):
         dir_ = os.getcwd()
 
     env = Environment(
+        autoescape=select_autoescape(
+            enabled_extensions=(),
+            default_for_string=False,
+            default=True),
```

Comment:
Sorry, but this change still does enable autoescape for all files (the default value should be `False`).
```suggestion
            default=False),
```

Reply:
In the context of this logic here, we only use Jinja2 to format local files e.g. `suite.rc`, so there is no untrusted tainted network stuff going on, and so there is no security issue at all.